cut-n-paste/synctex/synctex_parser.c

```diff
@@ -1090,7 +1090,7 @@ static void _synctex_display_boundary(synctex_node_t node) {
 #   define SYNCTEX_FILE (scanner->file)
 
 /*  Actually, the minimum buffer size is driven by integer and float parsing.
- *  �0.123456789e123
+ *  �0.123456789e123
```

Comment:
Why is this changed?

Reply:
Jesus christ, thanks. I guess it screwed up the old character? Anyways Atom likes to do things like this (like fix whitespaces automatically). I pretty much give up with nano so I don't have to worry about an IDE like Builder interfering to get around bad character propaganda but jesus